modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1011,6 +1011,44 @@ PERF_TEST_P_(SumPerfTest, TestPerformance)
     SANITY_CHECK_NOTHING();
 }
 
+//------------------------------------------------------------------------------
+#pragma push_macro("countNonZero")
+#undef countNonZero
```

Comment:
Why do we need this? Which configuration triggers this problem?

Reply:
@alalek, @mpashchenkov This is a workaround for the changes merged in this PR [#16657](https://github.com/opencv/opencv/pull/16657) supposed to ban any misuse of `countNonZero` function. I decided not to unban it but permit it for one particular test